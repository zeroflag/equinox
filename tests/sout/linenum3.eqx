"tests/mymod.eqx" need -> var mymod

mymod:w1 mymod:w2 + 3 =assert

: myword
  mymod:w1 mymod:w3 * ;

myword
