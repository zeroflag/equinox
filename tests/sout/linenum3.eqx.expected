  File "tests/sout/linenum3.eqx", line 6 
      005.  : myword
    =>006.    mymod:w1 mymod:w3 * ;
      007.  
  File "tests/sout/linenum3.eqx", line 8 
      007.  
